interactions/api/http/scheduledEvent.py

```diff
@@ -54,7 +54,7 @@ async def get_scheduled_event(
         guild_id, event_id = int(guild_id), int(guild_scheduled_event_id)
         params = {}
         if with_user_count:
-            params["with_user_count"] = with_user_count
+            params["with_user_count"] = f"{with_user_count}"
```

Comment:
Why f-string instead of string casting? Example `str(with_user_count)`
```suggestion
            params["with_user_count"] = str(with_user_count)
```